wire-ios-sync-engine/Source/Synchronization/EventProcessor.swift

```diff
@@ -161,7 +161,6 @@ actor EventProcessor: UpdateEventProcessor {
     }
 
     private func enqueueTask(_ block: @escaping @Sendable () async throws -> Void) async throws {
-        defer { processingTask = nil }
```

Comment:
question: why is this needed?

Reply:
This statement could lead to the first execution clearing the task value a second execution has already set, in my opinion.